Runtime/Interaction/HPUIEvents.cs

```diff
@@ -54,7 +54,7 @@ public HPUIHoverUpdateEventArgs(IHPUIInteractor interactorObject, Vector3 hoverP
         public Vector3 hoverPoint { get; set; }
 
         /// <summary>
-        /// The location of the attachTransform.
+        /// The location of the transform to attach to.
         /// <seealso cref="XRBaseInteractor.GetAttachTransform"/>
```

Comment:
Revert this. "attachTransform" has its own meaning in XRI

Reply:
done